src/locale.en.js

```diff
@@ -32,13 +32,16 @@ export const campsite = {
         here_now: 'I am here now',
         accessibility: 'Accessibility',
         facilities: 'Facilities',
-        facilities_options: {
-            full_service: 'Full Service ',
-            some: 'Some',
-            none: 'None',
-            permit: 'Permit',
+        price: 'Price',
+        price_options: {
+            free: 'Free',
             paid: 'Paid',
-            free: 'Free'
+            permit: 'Permit'
+        },
+        facilities_options: {
+            full_service: 'Full Service',
+            some_facilities: 'Some',
+            no_facilities: 'None'
```

Comment:
Above are the refactors I mentioned where I added `price` and cleaned up `facilities` to not include any pricing logic.